src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+
+  &__image-block {
+    margin: auto;
```

Comment:
The `&__image-block` element does not have any specific styles other than margin. If this is the only style it requires, consider whether the element is necessary, or if the margin can be applied to the `&__image` element directly.